FoodOrderApi/DataProvider/DbDataProvider.cs

```diff
@@ -0,0 +1,180 @@
+﻿using AutoMapper;
+using FoodOrderApi.Controllers;
+using FoodOrderApi.Model.Domain;
+using FoodOrderApi.Model.DTO;
+using FoodOrderApi.Repository;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ServiceStack;
+
+namespace FoodOrderApi.DataProvider
+{
+    public class DbDataProvider : Controller, IDataProvider
+    {
+        private readonly IDbProvider foodApiDbContext;
```

Comment:
All private variables should start with `_`. Please check all the other locations and update the same,

Reply:
Checked and updated all the private fields